### PR TITLE
Add predict alphafold domain

### PR DESCRIPTION
**Pull request by jpquast**
Added new `predict_alphafold_domain()` function to protti. 
Updated vignette to include description for the two new functions.

### CODE REVIEW

**Comment by jpquast** [comment]
Now also fixed a bug in `assign_missingness()`: Unequal number of replicates for comparisons are detected and correctly dealt with now.

**Review by dschust-r — CHANGES_REQUESTED** [review]
Added just two comments 🐨

File: R/fetch_mobidb.R [hunk 1]
```diff
@@ -1,116 +1,160 @@
-#' Fetch protein disorder information from MobiDB
+#' Fetch protein disorder and mobility information from MobiDB
 #'
-#' Fetches information about disordered protein regions from MobiDB.
+#' Fetches information about disordered and mobile protein regions from MobiDB.
```
Comment (dschust-r) [inline R/fetch_mobidb.R:3]:
```suggestion
#' Fetches information about disordered and flexible protein regions from MobiDB.
```

File: R/fetch_mobidb.R [hunk 2]
```diff
@@ -1,116 +1,160 @@
...
+#' should be fetched. This argument is mutually exclusive to the \code{organism_id} argument.
+#' @param organism_id optional, a character value providing the NCBI taxonomy identifier of an organism
+#' (TaxId) of an organism for which all available information should be retreived. This
+#' argument is mutually exclusive to the \code{uniprot_ids} argument.
+#' @param show_progress a logical value; if `TRUE` a progress bar will be shown.
+#' Default is `TRUE`.
 #'
-#' @return A data frame that contains start and end positions for disordered regions for each
-#' protein provided. The \code{feature} column contains information on the source of this
+#' @return A data frame that contains start and end positions for disordered and mobile protein
```
Comment (dschust-r) [inline R/fetch_mobidb.R:13]:
```suggestion
#' @return A data frame that contains start and end positions for disordered and flexible protein
```

**Review by dschust-r — CHANGES_REQUESTED** [review]
Minor comment, otherwise LGTM 🌾

File: R/assign_missingness.R [hunk 3]
```diff
@@ -145,26 +145,66 @@ from the conditions and assigned their missingness. The created comparisons are:
...
+    )
+  }
+  
+    unequal_replicates <- unequal_replicates %>% 
+    dplyr::distinct(.data$n_replicates, .data$comparison)
+
+  if (nrow(unequal_replicates) != 0) {
+    message("\n")
+    message(
+      strwrap("The following comparisons have been detected to have unqueal replicate numbers.
```
Comment (dschust-r) [inline R/assign_missingness.R:179]:
```suggestion
      strwrap("The following comparisons have been detected to have unequal replicate numbers.
```

**Review by dschust-r — CHANGES_REQUESTED** [review]
Found a typo 😮

**Review by dschust-r — APPROVED** [review]
LGTM 🐩

**Pull request merged**